docs/UserGuide.md

```diff
@@ -184,34 +184,15 @@ Examples:
 *  `tag-delete n/John Doe t/Adam and Steve` Removes the tag `Adam and Steve` from John Doe.
 *  `tag-delete n/Betsy Crower t/Lacy & Bacy t/Peter & Mary t/Jonny & Bonny` Removes the tags `Lacy & Bacy`, `Peter & Mary`, and `Jonny & Bonny` from Betsy Crower.
 
-### Locating Contacts by name: `find`
-
-You can quickly look up contacts by name and view all their details using `find`.
-
-Format: `find KEYWORD [MORE_KEYWORDS]`
-
-* `find` searches by `NAME` field.
-* `KEYWORD` is not case-sensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned.
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
-
-![find message](images/findMsg.png)
-
-Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-
 ### Filtering Contacts by Job: `filter`
 
-If you are looking for contacts with a specific job, you can use `filter` to filter out a list of contacts whose job matches your input.
+If you are looking for contacts with a specific name or job, you can use `filter` to filter out a list of contacts whose job matches your input.
```

Comment:
Perhaps can add (name or job) in the second line instead of just job

Reply:
oopsies :(